frontend/src/app/uploads/DeviceLookup/DeviceSearchResults.tsx

```diff
@@ -61,7 +64,21 @@ const DeviceSearchResults = (props: SearchResultsProps) => {
               <td id={`model-name-${idx}`}>{d.model}</td>
               <td id={`test-type-${idx}`}>
                 {d.supportedDiseaseTestPerformed
-                  ?.map((sd) => sd.supportedDisease.name)
+                  ?.reduce(
+                    (
+                      diseaseNames: Array<String>,
```

Comment:
Nitpick, I wonder if using a Set would be a better choice here

Reply:
I considered that, but we'd have to get the disease names into an array to `join` them anyhow so I just went with this.